xml/net_sync.xml

```diff
@@ -80,782 +74,369 @@
-       was possible to merge the differences in the local copy.
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term>P</term>
+     <term><option>-v</option></term>
      <listitem>
-      <para>
-       The local version was patched with the version on the server.
-      </para>
+      <para> Outputs more verbose text</para>
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term>C</term>
+     <term><option>-a</option></term>
      <listitem>
-      <para>
-       The local file conflicts with current version in the repository.
-      </para>
+      <para> Archive mode; copy files recursively and preserves timestamps,
+       user/group ownership, file permissions, and symbolic links. </para>
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term>?</term>
+     <term><option>-z</option></term>
      <listitem>
-      <para>
-       This file does not exist in CVS.
-      </para>
+      <para>Compress the transmitted data</para>
      </listitem>
     </varlistentry>
    </variablelist>
-   <para>
-    The status <literal>M</literal> indicates a locally modified file. Either
-    commit the local copy to the server or remove the local file and run the
-    update again. In this case, the missing file is retrieved from the server.
-    If you commit a locally modified file and the file was changed in the same
-    line and committed, you might get a conflict, indicated with
-    <literal>C</literal>.
+   <note>
+    <title>Trailing Slashes Count</title>
+    <para>
+     Rsync has a peculiar syntax when it comes to directory names: the
+     <emphasis>trailing slash</emphasis>. If you put a trailing slash after the
+    directory it means the <emphasis>content</emphasis> of that directory. If
+    you omit the slash, you mean the directory itself.
+   </para>
+   </note>
+  </sect2>
+  <sect2 xml:id="sec.net.sync.rsync.copy.local.files-and-dirs">
+   <title>Copying Files and Directories Locally</title>
+   <para> Assuming a directory <filename>/var/backup</filename>
+    with write permissions.
+    To copy a single file on your local machine from
+    one path to another path, use the following command: </para>
+   <screen>&prompt.user;<command>rsync</command> -avz backup.tar.xz /var/backup/</screen>
+   <para>The file <filename>backup.tar.xz</filename> is copied to
+     <filename>/var/backup/</filename>, the absolute path will be
+     <filename>/var/backup/backup.tar.xz</filename>.</para>
+   <para>
+    Keep in mind to insert the <emphasis>trailing slash</emphasis> after the
+    <filename>/var/backup/</filename> directory! If you do not insert the slash,
+    the file <filename>backup.tar.xz</filename> is copied to <filename>/var/backup</filename>
+    (file) <emphasis>not</emphasis> inside the directory <filename>/var/backup/</filename>!
+   </para>
+   <para>
+    To copy a directory is similar to copy a single file. The following
```

Comment:
To copy a directory is similar to copy a single file.

**Copying a directory is similar to copying a single file**